scripts/data_collection/repositories_table.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+from psycopg2 import sql
+
+from data_collection.db_connect import DatabaseConn
+
+
+class RepositoriesTable:
+    def __init__(self, conn: DatabaseConn):
+        self._conn = conn
+
+    def create(self):
+        if self._conn is not None:
+            self._conn.execute(
+                sql.SQL("""
+                create table if not exists kotlin_repositories_updates (
+                repository_id       serial      primary key,
+                username            varchar(64)   not null,
```

Comment:
I am not sure, but maybe should we create an enum class with table's fields find use the everywhere since if we will change this name now we have to change it in all places in the code and it can be difficult
What do you think?